specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -747,6 +747,143 @@
         }
       }
     },
+    "DaprResiliencyRetryPoliciesCollection": {
+      "description": "Dapr Resiliency Retry Policy ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprResiliencyRetryPolicy"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprResiliencyRetryPolicy": {
+      "description": "Resiliency Policy Retries",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "policy name",
+          "type": "string"
+        },
+        "policy": {
+          "description": "back-off and retry interval strategy",
+          "type": "string"
+        },
+        "duration": {
+          "description": "time interval between retries",
+          "type": "string"
+        },
+        "maxRetries": {
+          "description": "maximum number of retries to attempt",
+          "type": "string"
+        },
+        "maxInterval": {
+          "description": "determines the maximum interval between retries",
+          "type": "string"
+        }
+      }
+    },
+    "DaprResiliencyCircuitBreakerPoliciesCollection": {
+      "description": "Dapr Resiliency Circuit Breaker Policy ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaprResiliencyCircuitBreakerPolicy"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DaprResiliencyCircuitBreakerPolicy": {
```

Comment:
>DaprResiliencyCircuitBreakerPolicy [](http://example.com/codeflow?start=5&length=34)

needs ARM envelope atleast as a proxy resource

Reply:
do an allOf ProxyResource from the v4 common types

Reply:
Added